js/ui/validation_summary.js

```diff
@@ -217,53 +217,61 @@ var ValidationSummary = CollectionWidget.inherit({
             }
         });
 
-        that.option("items", items);
+        this.option("items", items);
     },
 
-    _itemValidationHandler: function(itemValidationResult) {
-        var items = this.option("items"),
-            isValid = itemValidationResult.isValid,
-            elementIndex,
-            replacementFound = false,
-            newMessage = itemValidationResult.brokenRule && itemValidationResult.brokenRule.message,
-            validator = itemValidationResult.validator;
+    _itemValidationHandler(itemValidationResult) {
```

Comment:
```suggestion
    _itemValidationHandler({ isValid, validator, brokenRules }) {
```
And remove L224-L226